python/mdvtools/dbutils/routes.py

```
@@ -5,147 +5,587 @@
-        try:
-            print("Creating project")
-            
-            # Get the next ID from the database
-            next_id = db.session.query(db.func.max(Project.id)).scalar()
-            if next_id is None:
-                next_id = 1
-            else:
-                next_id += 1
+def register_routes(app, ENABLE_AUTH):
+    from flask import request, jsonify, render_template
+    from mdvtools.dbutils.mdv_server_app import maybe_require_user, update_cache, active_projects_cache, user_project_cache,user_cache,user_project_cache, all_users_cache, active_projects_cache
```

Ruff (0.8.2)

Redefinition of unused `user_project_cache` from line 13

Remove definition: `user_project_cache`

(F811)